src/bin/cargo/commands/rustdoc.rs

```diff
@@ -48,20 +55,35 @@ pub fn cli() -> Command {
 
 pub fn exec(config: &mut Config, args: &ArgMatches) -> CliResult {
     let ws = args.workspace(config)?;
+    let output_format = if let Some(output_format) = args._value_of("output-format") {
+        config
+            .cli_unstable()
+            .fail_if_stable_opt("--output-format", 12103)?;
+        OutputFormat::from_str(output_format).unwrap()
```

Comment:
nit: `output_format.parse()?` here for simpilicity and can also remove the `use std::str::FromStr` at the top of the file?